ethcore/trace/src/types/trace.rs

```diff
@@ -122,6 +124,7 @@ impl From<ActionParams> for Create {
 			value: p.value.value(),
 			gas: p.gas,
 			init: p.code.map_or_else(Vec::new, |c| (*c).clone()),
+			create_type: p.call_type,
```

Comment:
This reads a bit awkwardly imo, "is it a create type or a call type?", so maybe it should be renamed `action_type` in `ActionParams`?